packages/sanity/src/core/bundles/components/dialog/BundleIconEditorPicker.tsx

```diff
@@ -75,18 +66,24 @@ export function BundleIconEditorPicker(props: {
       content={
         <Container data-testid="popover-content">
           <Flex gap={1} padding={1}>
-            {COLOR_HUES.map((hue) => (
-              <Button
-                key={hue}
-                mode="bleed"
-                onClick={handleHueChange(hue)}
-                padding={1}
-                selected={value.hue === hue}
-                data-testid={`hue-button-${hue}`}
-              >
-                <Avatar color={hue} size={0} />
-              </Button>
-            ))}
+            <TooltipDelayGroupProvider>
```

Comment:
Forgive me if this is being explained in a different part and I missed it: why are we adding this? Is it to improve performance / are we fixing something else? 🤔

Reply:
What this DelayGroupProvider does is adds a context in which all the tooltips that belong to the same context will share the "delay" state.
Example, you hover on the first item after 300ms you see the tooltip, you then hover on the second item, the tooltip is shown immediately, so you don't have to wait 300ms on each button to see the tooltip.

Reply:
Ahhh that's really neat!
Thank you!